content/en/contribute/code/workflow.md

```diff
@@ -173,9 +173,11 @@ Issues in this column have been prioritised and are ready for development. The i
 
 ### In progress
 
-Issues in this column are being actively worked on, which includes development, design, code reviews, and testing.
+Issues in this column are being actively worked on, which includes development, design, documentation, code reviews, and testing. A work is considered complete when the documentation and test coverage (unit tests, e2e tests or integration tests) are finalized and the reviewers have approved it.
 
-Any code should be in a branch in each of the repositories you update. The name of the branch should be in the form `<issue-number>-<readable-name>`, for example `1104-inclusive-export`. Follow the [Quality Assistance]({{< ref "contribute/medic/product-development-process/quality-assistance" >}}) process to take full ownership of what you are building.
+#### Involve quality assistance early
+
+Involve [Quality Assistance]({{< ref "contribute/medic/product-development-process/quality-assistance" >}}) from the start of the process to take full ownership of what you are building.
 
 Use the following template for QA feedback throughout the development.
```

Comment:
This was not part of your code changes, but I suggest moving this template under `Testing`, as this section here is about Project statuses and not so much about code review feedback or testing.

Reply:
Agree, this testing section was so confusing

Reply:
Still weird, but better. When we restructure the page, then we'll improve it more